docs/TUMOR_ONLY_SNV_BENCH_RESULTS.md

```diff
@@ -0,0 +1,109 @@
+# Tumor-Only SNV Benchmarking Methods and Results
+When it comes to somatic variant calling, having a matched normal sample from the patient is always preferred.
+However, this is not always possible.
+Mutect2, in our opinion as of this writing, is the best caller to use when a matched normal is not available in tumor-only calling mode.
+Using the same gold standard dataset that we used to benchmark (see the first two sections of [this doc](https://github.com/kids-first/kf-somatic-workflow/blob/feature/mb-add-bench-doc/docs/SOMATIC_SNV_BENCHMARK.md)) our somatic with matched tumor-normal variant calling methods, we reviewed filtering strategies from other publications ([TOSCA](https://academic.oup.com/bioinformaticsadvances/article/2/1/vbac070/6717791?login=false) and [HIVE Dragen Pipeline](https://www.fda.gov/science-research/fda-science-forum/hive-dragen-pipeline-enables-somatic-tumor-only-filtration)), and chose the best combination that maximized F1 score while minimizing true positive (TP) loss as aggressively pursuing false positive (FP) reduction and results in a massive loss of TP calls.
```

Comment:
Just making a note to update the link for `this doc` here. It's currently pointing to your branch which will be deleted. Now that you've merged we should be pointing to master.